less/home-page.less

```diff
@@ -14,6 +23,13 @@
   
   }
 
+  .top-image {
+    .border-radius (8px)
+    // border-radius: 8px;
+    margin-left: 2px;
+    margin-right: 2px;    
+  }
+
```

Comment:
you might want a media query in here on this image to give it a max-width when in phone size. It was large when I looked at it on my desktop (through that was in html preview which isn't always perfect).